### PR TITLE
Imported and injected redux into T4Loans. Created central store, pre-…

### PR DESCRIPTION
**Pull request by luisrivera1**
Redux Setup into T4Loans. In this commit you will find: 

- The redux and react-redux packages. 

- The initialization of the central store in index.js, which contains the state of the application.

- The injection of Redux into React with the <Provider tag.

- Pre-elimary reducers and actions only for example purposes.

- Root Reducer which combines all reducers, which is imported into the index.js as pass into the store as a parameter.

*Important* No implementation was started for a specific class, all coments found are for example purposes. If you which to visualize the initialstate, you can console.log(store.state); in the index.js file.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -19,22 +19,37 @@ import ReactDOM from "react-dom";
...
 import "assets/scss/material-dashboard-pro-react.scss?v=1.8.0";
 
 const hist = createBrowserHistory();
 
+
+const store = createStore(rootReducer);
+
+
 ReactDOM.render(
+    <Provider store={store} >
```
Comment (cfboy) [inline src/index.js:45]: Indent this block of code.

File: src/index.js [hunk 2]
```diff
@@ -19,22 +19,37 @@ import ReactDOM from "react-dom";
...
   <Router history={hist}>
     <Switch>
       <Route path="/rtl" component={RtlLayout} />
       <Route path="/auth" component={AuthLayout} />
       <Route path="/admin" component={AdminLayout} />
       <Redirect from="/" to="/admin/dashboard" />
     </Switch>
-  </Router>,
+  </Router> </Provider>,
```
Comment (cfboy) [inline src/index.js:53]: Indent.

Comment (luisrivera1) [reply src/index.js]: done

File: src/index.js (hunk 1 above)
Comment (luisrivera1) [reply src/index.js]: done

File: src/store/reducers/customer.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import * as actionsTypes from '../action'
+const initialState = {
+    customer: [
+        {id: '1', firstname: "Luis", initial: 'A', lastname: 'Rivera', secondlastname: 'Marquez', birthday:'08/31/1995', gender: 'Male', phonenumber: '7876244804', address: 'Calle Piedras Negras', city:'San Juan',
+        state: 'PR', zipcode: '00926'},
+        {id: '2', firstname: "Cristian", initial: 'F', lastname: 'Torres', secondlastname: 'Alvarado' ,birthday:'08/31/1995', gender: 'Male', phonenumber: '7876244804', address: 'Calle Piedras Negras', city:'San Juan',
+            state: 'PR', zipcode: '00926'},
+        {id: '3', firstname: "Alejandro", initial: 'A', lastname: 'Reyes', birthday:'08/31/1995', gender: 'Male', phonenumber: '7876244804', address: 'Calle Piedras Negras', city:'San Juan',
+            state: 'PR', zipcode: '00926'},
+        {id: '3', firstname: "Christian", initial: ' ', lastname: 'Perez', secondlastmane:'Villanueva', birthday:'08/31/1995', gender: 'Male', phonenumber: '7876244804', address: 'Calle Piedras Negras', city:'San Juan',
```
Comment (alejoreyes96) [inline src/store/reducers/customer.js:10]: I know all this is dummy data so most of it is duplicated, but you should change the duplicate _id_.

**Pull request merged**